lib/workers/repository/dependency-dashboard.ts

```diff
@@ -245,6 +266,25 @@ export async function ensureDependencyDashboard(
 
   issueBody = appendRepoProblems(config, issueBody);
 
+  if (hasDeprecations) {
+    issueBody += '> ⚠ **Warning**\n> \n';
+    issueBody += 'The following dependencies are deprecated:\n\n';
```

Comment:
```suggestion
    issueBody += 'These dependencies are deprecated:\n\n';
```

Could save some extra characters here.